README.md

```
@@ -52,7 +52,7 @@ Go to browser and open the link below to access REST API docs
 
 And using the link below you can access the GraphQL documentation
 
-> https://localhost/api/graphql/graphql_playground
+> https://localhost/api/graphql
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)